Formula/vis.rb

```diff
@@ -0,0 +1,54 @@
+class Vis < Formula
+  desc "Vim-like text editor"
+  homepage "https://github.com/martanne/vis"
+  url "https://github.com/martanne/vis/archive/v0.4.tar.gz"
+  sha256 "f11ba41cfb86dd39475960abfd12469de4da0ccfdb941f1d7680d89d987694c5"
+  head "https://github.com/martanne/vis.git"
+
+  depends_on "lua" => :build
```

Comment:
This seems unlikely to be true? IIRC, installed Luarocks still require the `lua` interpreter around.